labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise02.scala

```diff
@@ -11,15 +11,17 @@ object ListManipulationExercise02 {
    * As usual, various ways exist: pattern matching, folding, ...
    */
   def maxElementInList(l: List[Int]): Int = {
-    error("fix me")
+    l.reduceLeft( _ max _ )
   }
 
   /**
    * Calculate the sum of the equally position elements
-   * of the two list
+   * of the two lists
    */
   def sumOfTwo(l1: List[Int], l2: List[Int]): List[Int] = {
-    error("fix me")
+    (if (l1.length > 0 && l2.length > 0) (l1, l2).zipped.map(_+_)
+    else if(l1.length == 0) l2
+    else l1)
```

Comment:
I'm not sure if this has the right behavior in all cases.
What about `[1]` and `[1, 2, 3]`?

It seems that the answer should be `[2, 2, 3]`, but your code does not give that